qt/scientific_interfaces/Inelastic/Processor/SymmetrisePresenter.cpp

```diff
@@ -136,14 +142,17 @@ void SymmetrisePresenter::handleDoubleValueChanged(std::string const &propName,
   }
 }
 
+void SymmetrisePresenter::handlePreviewClicked() {
+  setIsPreview(true);
+  runTab();
+}
 void SymmetrisePresenter::handleDataReady(std::string const &dataName) {
-  if (m_view->validate()) {
+  if (validate()) {
```

Comment:
I think we might want to call `m_runPresenter->validate()` rather than the `validate` method from the parent class

Reply:
`m_runPresenter` is a private member of the parent class. So that's why I reimplemented validate in the parent class, to be able to call `m_runPresenter` validate indirectly. Do you think there is another way to do it ?

Reply:
Ah I see. Another way might be to put `m_runPresenter` in the protected section so that classes that inherit from the parent can access it, if that sounds good to you?